app/assets/stylesheets/ui-components/jquery-ui.scss.erb

```diff
@@ -902,19 +902,19 @@ body .ui-tooltip {
 .ui-widget-content {
   border: 1px solid #dddddd;
   background: #ffffff;
-  color: $application-text-color;
+  color: #487ba2;
```

Comment:
we should be using a variable here and throughout this file, not a hard-coded value